src/common/notfound/NotFound.tsx

```diff
@@ -0,0 +1,49 @@
+import { Container, Heading, Flex, Text, Divider } from '@chakra-ui/layout';
+import { Button, Box } from '@chakra-ui/react';
+import { HiOutlineCalendar } from 'react-icons/hi';
+import { Link } from 'react-router-dom';
+
+import { useDarkMode } from 'lib/hooks/useDarkMode';
+import { getReadableTerm } from 'lib/utils/terms';
+
+type Props = {
+  item: string;
+  term: string;
+  timetableButton?: boolean;
+  timetable?: boolean;
+};
+
+export function NotFound({ item, term, timetable }: Props) {
```

Comment:
```suggestion
export function NotFound({ children, term, timetable }: PropsWithChildren<Props>) {
```
Let's replace `item` with `children` using `PropsWithChildren`. There are a few examples in the codebase that use this, but essentially this will allow us to use the component like:
```jsx
<NotFound term="202109">
   Unable to find saved courses from your
</NotFound>
```